package com.nebulabyte.agent.apollo_backend.Dto;
import java.util.List;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class SearchResultDto {

    private String query;
    private int totalMatches;

    private List<CitationDto> matches;

}



